src/internal/operators/bufferTime.ts

```diff
@@ -231,23 +239,26 @@ function dispatchBufferTimeSpanOnly(this: SchedulerAction<any>, state: any) {
     state.context = subscriber.openContext();
     state.context.closeAction = this.schedule(state, state.bufferTimeSpan);
   }
+
+  state.subscriber.destination.next(prevContext.buffer);
 }
 
 function dispatchBufferCreation<T>(this: SchedulerAction<DispatchCreateArg<T>>, state: DispatchCreateArg<T>) {
```

Comment:
If these were members of the class instead of functions that have been bifurcated, they could also access the private `destination` member of the instance & wouldn't need it passed in.

I feel like there's some good reason why the scheduler was written this way, but it feels like a code smell to me, being naive about what those reasons may be.